__tests__/app.test.js

```diff
@@ -115,3 +117,59 @@ describe("Error Handling 404 ", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("return 200 status, should get a comment by its article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(Array.isArray(comments)).toBe(true);
+        expect(comments.length).toBeGreaterThan(0)
+        expect(comments).toBeSorted({descending: true})
+        comments.forEach((comment) =>{
+          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+           
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            
+        })
+    
+        });
+      });
+  });
+  test('return 200 status, should return an error message when no comments are found', () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ comments })=> {
+        expect(comments).not.toBe([])
```

Comment:
I think you may have misunderstood Alice's previous feedback. What we're looking for here is a 200 test for articles that have no comments, and we're expecting an array to come back. Here, you're just testing that there are comments on article 1.